src/ILCompiler/src/Program.cs

```diff
@@ -302,7 +304,11 @@ private int Run(string[] args)
                         entrypointModule = module;
                     }
 
-                    compilationRoots.Add(new ExportedMethodsRootProvider(module));
+                    // TODO: Wasm fails to compile some of the exported methods due to missing opcodes
+                    if (!_isWasmCodegen)
```

Comment:
This coming back looks like a bad merge.

Reply:
Thanks for this. Been trying to fix the Windows build error